packages/perseus/src/widgets/interactive-graphs/types.ts

```diff
@@ -74,3 +74,9 @@ export interface CircleGraphState extends InteractiveGraphStateCommon {
 }
 
 export type PairOfPoints = [Coord, Coord];
+
```

Comment:
I moved this type here as I noticed it was being used across several different files